src/app/prevention/stop-the-breed/game-config.ts

```diff
@@ -0,0 +1,79 @@
+import { Item } from './models/Item';
+import { Element } from './models/Element';
+
+// For the background of the game
+
+export const backgroundConfig: Item = {
+  imageURL: '../../../assets/img/stop-the-breed/background_village.png',
+  positionX: 0,
+  positionY: 0,
```

Comment:
Maintain a configuration file where these co-ordinates are defined. It will be easy to change when in the file when you want to change. Do not hard code.

Reply:
not just coordinates, the url and everything which can be changed.

Reply:
@shreyans29 this is itself the game-config file. I made this for that purpose only. Do I need to make a config file for this as well?